index.html

```diff
@@ -560,15 +564,18 @@
         <div dfn-for="MediaTrackSettings">
           <p>
             The <dfn><code>focalLength</code></dfn> dictionary member
-            represents the <a>depth map</a>'s <a>focal length</a>.
+            represents the <a>depth map</a>'s <a>focal length</a>. It is a
+            <a>non-constrainable property</a>.
```

Comment:
I copied the name from #102 :)

Would rename of the concept into "unchangeable property" make things better? I'm happy to refactor the PR further given concrete suggestions how to convey the read-only-ness of these in the context of the depth stream tracks. Currently, those constraints do not exist for the regular MSTs, but could be if upstreamed to a future version of the mediacapture-main.